skimage/transform/_warps.py

```diff
@@ -172,12 +174,12 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
         tform.params[0, 1] = 0
         tform.params[1, 0] = 0
 
-        out = warp(image, tform, output_shape=output_shape, order=order,
+        out = warp(img_in, tform, output_shape=output_shape, order=order,
```

Comment:
The proposed solution doesn't work in this case since the clipping is made in `warp` :confused:

Reply:
Thanks for pointing this out ! Fixed it in fallback code by performing clipping outside of warp function :smiley:.

Reply:
Excellent idea! The `_clip_warp_output` call can be performed outside the `if ... elif ... else ...` block once for all now :wink: